<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>475</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>335</width>
    <height>475</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>335</width>
    <height>475</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Montserrat Medium</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Лабораторная работа 4</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QRadioButton" name="radioButton_random">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>88</y>
      <width>253</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Случайные значения</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_input">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>166</y>
      <width>160</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ручной ввод</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_show">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Построить</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_x_max">
    <property name="geometry">
     <rect>
      <x>106</x>
      <y>48</y>
      <width>211</width>
      <height>33</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-99.989999999999995</double>
    </property>
    <property name="singleStep">
     <double>0.500000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_x_min">
    <property name="geometry">
     <rect>
      <x>106</x>
      <y>8</y>
      <width>211</width>
      <height>33</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-99.989999999999995</double>
    </property>
    <property name="singleStep">
     <double>0.500000000000000</double>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_args">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>277</y>
      <width>301</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_n">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>106</x>
      <y>126</y>
      <width>211</width>
      <height>33</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="accelerated">
     <bool>false</bool>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="value">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>8</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>x_min = </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>48</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>x_max = </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>126</y>
      <width>36</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>n = </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>204</y>
      <width>309</width>
      <height>66</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Введите через пробел &lt;/p&gt;&lt;p&gt;точки в заданном диапазоне&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>от</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>350</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>до</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_left">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>350</y>
      <width>101</width>
      <height>33</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-99.989999999999995</double>
    </property>
    <property name="singleStep">
     <double>0.500000000000000</double>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_right">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>350</y>
      <width>101</width>
      <height>33</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-99.989999999999995</double>
    </property>
    <property name="singleStep">
     <double>0.500000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>309</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Интервал построения графика&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>335</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
